pkg/splunk/enterprise/clustermaster.go

```diff
@@ -209,6 +209,9 @@ func getClusterMasterStatefulSet(client splcommon.ControllerClient, cr *enterpri
 		setupInitContainer(&ss.Spec.Template, cr.Spec.Image, cr.Spec.ImagePullPolicy, commandForCMSmartstore)
 	}
 
+	// Setup App framework init containers
```

Comment:
Any logical reason for the `setupAppInitContainers` to be part of  `getClusterMasterStatefulSet`, rather than being explicitly called in `ApplyClusterMaster`?  (similarly for other cases too)

Reply:
I placed `setupAppInitContainers()` here because the init container config is in the stateful set.  Also `setupInitContainer()` is also in this function, so I felt this was the best place.  We could move this to the `ApplyClusterMaster()` functions if we feel thats the best place for it.

Reply:
Thanks for clarifying, @jryb . Lets keep it here for now